### PR TITLE
V0.3 updated UI tests - with all 3 types of tasks

### PR DESCRIPTION
**Pull request by mattheuslee**

### CODE REVIEW

File: src/main/java/seedu/multitasky/model/util/EntryBuilder.java [hunk 1]
```diff
@@ -82,19 +84,17 @@ public EntryBuilder withEndDateAndTime(Calendar endDateAndTime) {
 
     public Entry build() throws Exception {
         if (startDateAndTime == null) {
-            // Floating task
```
Comment (akd13) [inline src/main/java/seedu/multitasky/model/util/EntryBuilder.java:85]: Why did you remove these markers?

Comment (mattheuslee) [reply src/main/java/seedu/multitasky/model/util/EntryBuilder.java:85]: I felt like the constructor names kinda explained what was happening already, and also the comments weren't lining up with the lines they were supposed to be describing.

**Review by akd13 — APPROVED** [review]

**Review by ChuaPingChan — APPROVED** [review]

**Review by kevinLamKB — APPROVED** [review]
